src/studio/nuxeo-studio/working-in-studio/branding.md

```diff
@@ -391,168 +392,95 @@ history:
-
-![]({{file name='-blue_background_style.png'}} ?w=300,border=true,thumbnail=true)
-
-</td><td colspan="1">![]({{file name='blue_background_container.png'}} ?w=300,border=true,thumbnail=true)</td></tr><tr><td colspan="1">
-
-Define a class in the widget editor.
-
-</td><td colspan="1">Declare your blue background in the CSS Editor.
-
-</td><td colspan="1">Enjoy your blue container.</td></tr></tbody></table></div>
+{{#> callout type='warning'}} While selecting your background, be careful not to use heavy files to ensure a quick display to your users.
+{{/callout}}
 
-### Overriding an Existing Rule
+### Selecting a Background Image  
 
-You want to change an existing style to fit your needs? In addition to the [Nuxeo UI Style Guide](http://showcase.nuxeo.com/nuxeo/styleGuide/) that explains the default set of CSS existing rules, you will need to use a browser's web inspector to understand what to override.
+1.  In `Background Image`, click on **Select Resources**.
+1.  Define how the image should be displayed using pixels or the position attribute.
+1.  Indicate if the background covers all the screen (with CSS3 / recent browsers).
+1.  Pick a color that will be displayed if the background image is too small and not covering all the screen.
 
-Let's display the main title of pages in pink:
+|  Before | Studio Configuration | After |
+|---|---|---|
+| ![]({{file name='branding_picture_nuxeo.png'}} ?w=300,border=true) | ![]({{file name='login_customization.png'}} ?w=300,border=true)  |![]({{file name='branding_picture.png'}} ?w=300,border=true)  |
 
-<div class="table-scroll"><table class="hover"><tbody><tr><th colspan="1">1\. Inspect the existing style</th><th colspan="1">2\. Declare your new rule</th><th colspan="1">3\. See the result</th></tr><tr><td colspan="1">![]({{file name='override_inspect.png'}} ?w=300,border=true,thumbnail=true)</td><td colspan="1">![]({{file name='disable_branding.png'}} ?w=300,border=true,thumbnail=true)</td><td colspan="1">![]({{file name='override_result.png'}} ?w=300,border=true,thumbnail=true)</td></tr><tr><td colspan="1">
+### Selecting a Background Video
 
-The pink arrows show the main title at the top of the page, the code and its CSS declaration are at the bottom, in the web inspector. Seeing this, you will need to override the `default.CSS****` `h1` rule that contains the color declaration.
+1.  Select a background video.
+1.  In `Background Video URLs / MIME Type`, click **Add**.
+1.  Enter the URL and MIME Type of your video.
+1.  Define the options for the video, such as looping or muting the video.
 
-You can use the parent CSS classes, underlined here in green, to have a stronger CSS selector than the default one.
+## Customizing the Login Box
 
-</td><td colspan="1">
+### Selecting a Logo
 
-Your rule contains three selectors: `.content`, .`mainFragment` and `h1`.
+By default, the logo used is the one defined as your Application Logo. You can select another logo to appear in your login box.
```

Comment:
Application logo is not chosen unless you are using JSF UI, so I'd recommend saying: 
By default, the logo used is the one defined as your Application Logo if you are using JSF UI. If you are using Web UI only, no application logo can be defined and you should keep the related option to no.